### PR TITLE
build(release): add E2E release notes for Kubera 2.0

### PR DESCRIPTION
**Pull request by AmitKumarDas**
Signed-off-by: AmitKumarDas <amit.das@mayadata.io>

### CODE REVIEW

File: testplan/docs/kubera-e2e-release-2.0.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        AccessID:      awsAccessID,
+        SecretKey:     awsSecretKey,
+    },
+    "aws bucket with duplicate name": {
+        CloudProvider: "aws",
+        Name:          "aws-dmaas-test",
+        AccessID:      awsAccessID,
+        SecretKey:     awsSecretKey,
+    },
+    "aws bucket with name length less then 6 character": {
```
Comment (harshshekhar15) [inline testplan/docs/kubera-e2e-release-2.0.md:31]: typo - `than`

File: testplan/docs/kubera-e2e-release-2.0.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        AccessID:      cloudianAccessID,
+        SecretKey:     cloudianSecretKey,
+    },
+    "cloudian bucket with name length more then 25 character": {
+        CloudProvider: "cloudian",
+        Name:          "cloudian-dmaas-test-backup-restore-cred",
+        AccessID:      cloudianAccessID,
+        SecretKey:     cloudianSecretKey,
+    },
+    "cloudian bucket withot name": {
```
Comment (harshshekhar15) [inline testplan/docs/kubera-e2e-release-2.0.md:89]: typo - `without`

**Review by prabhatkumarthakur — APPROVED** [review]
lgtm

**Review by gprasath — APPROVED** [review]

**Review by amitbhatt818 — APPROVED** [review]

File: testplan/docs/kubera-e2e-release-2.0.md (hunk 1 above)
Comment (AmitKumarDas) [reply testplan/docs/kubera-e2e-release-2.0.md:31]: thanks

File: testplan/docs/kubera-e2e-release-2.0.md (hunk 2 above)
Comment (AmitKumarDas) [reply testplan/docs/kubera-e2e-release-2.0.md:89]: thanks. will correct.

**Review by rohan2794 — APPROVED** [review]

**Pull request merged**